settings.gradle

```diff
@@ -20,3 +20,6 @@ project(":frameworks/kafka").name = "kafka"
 
 include 'frameworks/hdfs'
 project(":frameworks/hdfs").name = "hdfs"
+
+include 'frameworks/template'
```

Comment:
Should this be here? Presumably someone adding a framework would add these lines when they actually create a new one in the repository, but doesn't seem worth including it by default.